src/pages/examSetter/Grading.tsx

```
@@ -0,0 +1,141 @@
+        visible={isModalVisible}
+        onCancel={() => setIsModalVisible(false)}
+        footer={null}
+      >
+        <List
+          itemLayout="horizontal"
+          dataSource={candidates}
+          renderItem={(candidate: Candidate) => (
+            <List.Item
+              actions={[
+                candidate.graded ? (
+                  <Tooltip title="Already Graded">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)